contributing_to_docs/doc_guidelines.adoc

```diff
@@ -746,18 +748,36 @@ There might be advanced networking examples that require specific IP addresses,
 
-In OpenShift docs:
+In OpenShift Container Platform docs:
 
-* All links to internal content is created using `xref` and **must have an anchor ID**.
+* All links to internal content is created using `xref` and each `xref` **must have an anchor ID**.
 * Only use `xref` in assemblies, not in modules.
-* All links to external websites are created using `link`.
-* Links between different distros, such as from ROSA to OpenShift Container Platform, are external links and not cross-references. These external links use `link` instead of `xref` and need to be in distro-specific files or conditionalized to apply to the relevant distros.
+* All links to external websites are created by using `link`.
+* Links between different distributions, or _distros_, such as from ROSA to OpenShift Container Platform, are external links and not cross-references. These external links use `link` instead of `xref` and must be in distro-specific files or a conditionalized state to apply to the relevant distros.
 
 [IMPORTANT]
 ====
-Do not split link paths across lines when wrapping text. This will cause issues with the doc builds.
+Do not split link paths across lines when wrapping text. This will cause issues with the documentation builds.
 ====
 
+[id="lead-in-link-sentences"]
+=== Lead-in link sentences
+
+To maintain consistency across OpenShift Container Platform documentation, ensure that you adhere to the following guidance for lead-in link sentences:
+
+* Maintain the `For more information, see "<section_name>."` format when pointing to a section in the same document.
+* Wrap the `<section_name>` string in quotation marks. The section name can be the title of a module or an assembly.
+* Do not state "in the Additional Resources section" or other variants. Mentioning the `<section_name>` is enough to prompt a documentation user to find the section in the same document.
+* When pointing to a link that exists in another customer document, use the `See "<section_name>" in _<document_name>_.` format. Note the quotation marks around `<section_name>` and the italics around `<document_name>`.
+* You can add link sentences to either the _Additional resources_ section or the _Next steps_ section, but know the following differences:
+** Additional resources - module level: Requires the `.Additional resources` section heading. The listed resources link to other material closely related to the contents of the module. Focus on relevant resources that might interest the user. Do not list resources for completeness.
+** Additional resources - assembly level: Requires the `== Additional resources` section heading. Similar to the previous item, but if a resource applies to all of the modules in a user story, consider listing the resource in the _Additional resources_ section of the assembly file. If you do this, consider mentioning the resource in the module as a comment.
```

Comment:
```suggestion
** Additional resources - assembly level: Requires the `== Additional resources` section heading. If a resource applies to all of the modules in a user story, consider listing the resource in the _Additional resources_ section of the assembly file. If you do this, consider mentioning the resource in the module as a comment.
```

Reply:
I would suggest not relating this to a preceding bullet. Let it be its own thing, even if it is similar.